playbooks/scenario1_5_setup.yml

```diff
@@ -51,3 +58,85 @@
           - listen: "80"
             server_name: "localhost"
             extra_parameters: "{{ extra_parameters }}"
+
+- name: Copy and configure test script files to Bastion
+  hosts: bastion
+  vars:
+    test_path: /home/linux/test
+    project_path: /home/ansideln/work/customer-service-monitoring
+    keys_path: "{{ project_path }}/scenarios/scenario1_5"
+    inventory_path: "{{ test_path }}/inventory/prod"
+    loadbalancer_public_ip: "{{ lookup('env', 'LOADBALANCER_PUBLIC_IP') }}"
+    verion: "HEAD"
```

Comment:
is it ok `verion` ?

Reply:
yep, pointer to the last commit in branch

Reply:
fixed